images/openstack-resource-controller.yml

```diff
@@ -0,0 +1,30 @@
+content:
```

Comment:
please double check the names with what we merged in org: https://gitlab.cee.redhat.com/hybrid-platforms/org/-/merge_requests/45/diffs#64cb79d1624d0229ed452d2b553b872c8f5777d0_0_7